ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -918,6 +918,24 @@ AHU_DFSS_CHWDT_DTM_RTC_RWISOVPC_OAFC_FDPSM2X:
   - OAFC
   - FDPSM2X
 
+AHU_SFSS_SFVSC_EFSS_EFVSC_HTWHLZTC_HWZTC_CHWZTC_SSPC_RSPC_CO2DSPMADC_MTM_FDPM3X:
+  description: "Single zone AHU."
```

Comment:
You are calling this a single-zone unit but you are using supply fan abstract types, which indicates this is a multi-zone unit. Please rectify this.

Reply:
Ok makes sense, please note this in the type description. Something like "AHU that serves multiple dampers all feeding a single zone."